web/src/pages/index.js

```diff
@@ -4,12 +4,15 @@ import PropTypes from 'prop-types';
 
 import Layout from '../components/layout';
 import Hero from '../components/hero';
+import SubHero from '../components/subHero';
 
 export default function IndexPage({ data }) {
-    const { heroImage } = data;
+    const { heroImage, plane } = data;
     return (
         <Layout>
+            <h1>index page</h1>
             <Hero image={heroImage} />
+            <SubHero image={plane} />
```

Comment:
Since this component is going to be used in multiple places I think we should come up with a better name. I'm open to whatever but it can be as simple as `BGImageWithCard`